ext/hal/microchip/mec/mec1501/MEC1501hsz.h

```diff
@@ -569,9 +569,9 @@ typedef enum IRQn {
 #define ESPI_MEM_BM_REGS  ((ESPI_MEM_BM_Type *)(ESPI_MEM_BM_BASE))
 
 /* eSPI Virtual Wire registers in IO component */
-#define ESPI_IO_VW_REGS      ((ESPI_IO_VW_Type *) ESPI_IO_VW_BASE)
+#define ESPI_IO_VW_REGS      ((ESPI_IO_VW_Type *) (ESPI_IO_VW_BASE))
```

Comment:
Are these change coming from an update of the microchip ext/hal or are these manual edition? 

If it's the later, an future update of the ext/hal will erase these.

I don't have a solution on how to act here though.